app/views/users/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= render partial: 'users/user', locals:{user:@user} -%>
+<div class="user-bio">
+  <label class="bio-label">bio</label>
+  <p><%= @user.bio -%></p>
+</div>
+<h3>Posts</h3>
+<% @user.posts.each do |post| %> <%= render partial: 'posts/post',
+locals:{post:post} -%> <% end %>
+<button class="show-all-posts-button">Show all posts</button>
```

Comment:
- [ ] When I click on this button, it does not redirect to the page that shows all the users posts. Please make sure that when a user clicks on this button, you are redirected to a page that looks similar to that seen in this [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png)